### PR TITLE
Address client-hints-04 feedback

### PR DESCRIPTION
**Pull request by igrigorik**
See https://github.com/httpwg/http-extensions/issues/359, https://github.com/httpwg/http-extensions/issues/360.

### CODE REVIEW

**Comment by mnot** [comment]
Oh, one thing -- you removed Downlink. It'd be good to mention that in the change notes, and on list, so people are aware.

File: draft-ietf-httpbis-client-hints.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Client Hints does not supersede or replace the User-Agent header field. Existing
 
 ## Notational Conventions
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in {{RFC2119}}.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHOULD", "SHOULD NOT", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 {{RFC2119}} when, and only when, they appear capitalized, as shown.
```
Comment (reschke) [inline draft-ietf-httpbis-client-hints.md]: This doesn't seem to be consistent with https://tools.ietf.org/html/rfc8174#section-2

Comment (igrigorik) [inline draft-ietf-httpbis-client-hints.md]: Good catch, looks like I took the text from outdated draft.

**Comment by igrigorik** [comment]
Thanks guys, pushed a few more updates. 

@reschke everything else looks good? If so I'll squash+push a new draft.

**Comment by igrigorik** [comment]
@reschke can you take another pass, please? Want to make sure we caught all of your feedback here.

**Comment by igrigorik** [comment]
@reschke merging. If you have any followup feedback, we can open a new issue.

**Pull request merged**